packages/sdk/src/modules/moduleFromEndpoints/types.ts

```diff
@@ -125,6 +130,23 @@ export type Options = {
    * Default request config for each request.
    */
   defaultRequestConfig?: RequestConfig;
+
+  /**
+   * Custom error handler for the requests.
+   *
+   * It's optional and it will use the default error handler if it's not provided.
+   *
+   * @example
+   * ```ts
+   * const options: Options = {
+   *   apiUrl: "https://api.example.com",
+   *   errorHandler: (error) => {
+   *     return refreshAndRetry(error);
```

Comment:
what is `refreshAndRetry`? if this is just an example, I don't think that its possible to retry when just an error is known

Reply:
@jagoral it's possible, we did it in the SAPCC but anyway it's just the example :)